src/utiltest.cpp

```diff
@@ -264,6 +264,10 @@ const Consensus::Params& RegtestActivateCanopy(bool updatePow, int canopyActivat
     return Params().GetConsensus();
 }
 
+const Consensus::Params& RegtestActivateCanopy() {
+    return RegtestActivateCanopy(false, Consensus::NetworkUpgrade::ALWAYS_ACTIVE);
```

Comment:
I thought we could only activate upgrades past Overwinter at height 1? @str4d?

Reply:
i saw a few examples using 
```C++
RegtestActivateHeartwood(false, Consensus::NetworkUpgrade::ALWAYS_ACTIVE);
```

Reply:
IIRC the issue with activating updates past genesis only affects regtest, or other modes where you rely on the hard-coded genesis blocks. For unit tests (which either don't build a chain, or build a fake chain with a newly-generated genesis block), this is fine.